### DISCUSSION TITLE
What unusual item sold for $3.26 on eBay in 2000?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#806] [Galaxy Trivia] [answered]
**Bonus:** Find a link to the eBay lot.

**Comment by peterbaccaro** [comment, marked as answer]
In 2000 a man from North Carolina discovered the **meaning of life** and decided to sell it on eBay. Curiously, it was worth only $3.26.

The seller wrote, “I have discovered the reason for our existence and will be happy to share this information with the highest bidder.” And apparently it was a popular item, as eight buyers were curious enough to put in offers. 

http://www.whattheheck.com/ebay/meaninglife.html

**Comment by SANKARAMDAS** [comment]
In 2000 a man from North Carolina discovered the meaning of life and decided to sell it on eBay. Curiously, it was worth only $3.26.
